crates/sui-rpc-api/proto/sui/rpc/v2alpha/owner.proto

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+// Enum of different types of ownership for an object.
+message Owner {
+  optional OwnerKind kind = 1;
+
+  optional string address = 2;
+  optional string object_id = 3;
+  optional uint64 version = 4;
+
+  enum OwnerKind {
+    OWNER_KIND_UNKNOWN = 0;
+    OWNER_KIND_ADDRESS = 1;
+    OWNER_KIND_OBJECT = 2;
+    OWNER_KIND_SHARED = 3;
+    OWNER_KIND_IMMUTABLE = 4;
+  }
```

Comment:
`OWNER_KIND_CONSENSUS_V2`?

Reply:
Yes eventually. It hasn't been released to devnet/testnet/mainnet yet so I opted to skip for now.